pkg/artifactory/datasource/repository/virtual/datasource_artifactory_virtual_repository_test.go

```diff
@@ -270,14 +271,14 @@ func mkNewVirtualTestCase(packageType string, t *testing.T, extraFields map[stri
 		CheckDestroy:             acctest.VerifyDeleted(t, fqrn, "", acctest.CheckRepo),
 		Steps: []resource.TestStep{
 			{
-				Config:           config,
-				Check:            resource.ComposeTestCheckFunc(checks...),
-				ConfigPlanChecks: testutil.ConfigPlanChecks(""),
-				// ConfigPlanChecks: resource.ConfigPlanChecks{
-				// 	PostApplyPreRefresh: []plancheck.PlanCheck{
-				// 		plancheck.ExpectEmptyPlan(),
-				// 	},
-				// },
+				Config: config,
+				Check:  resource.ComposeTestCheckFunc(checks...),
+				// ConfigPlanChecks: testutil.ConfigPlanChecks(""),
```

Comment:
Do we keep the commented-out code?

Reply:
Generally no, but I keep this so we can easily switch between the 2 different ways for checking plan diffs.